src/annotator/guest.js

```diff
@@ -146,16 +145,19 @@ export default class Guest {
 
     // Setup the document type-specific integration consisting of metadata extraction,
     // anchoring module and logic to respond to activity (eg. scrolling) in the page.
-    //
-    // In future the plan is to put these behind a common interface so that the
-    // `Guest` class doesn't need different code paths depending on the document type.
-
-    // nb. The `anchoring` field defaults to HTML anchoring and in PDFs is replaced
-    // by `PDFIntegration` below.
-    this.anchoring = anchoring;
-    this.documentMeta = new DocumentMeta();
     if (config.documentType === 'pdf') {
-      this.pdfIntegration = new PDFIntegration(this);
+      this.integration = new PDFIntegration(this);
+    } else {
+      const documentMeta = new DocumentMeta();
+      this.integration = {
+        anchor: htmlAnchoring.anchor,
```

Comment:
You could `import { anchor, describe } from './anchoring/html';` and then use 
```suggestion
        anchor,
```
Same for `describe`. 

The import above would be more consistent with the way you import from `anchoring/pdf` below.

Reply:
I agree with this change, but I think it will make more sense once the HTML integration has been extracted in a follow-up PR, where it will be obvious that `anchor` and `describe` are about HTML anchoring.